app/controllers/links_controller.rb

```diff
@@ -10,8 +10,11 @@ def index
 	end
 
 	def create
+		#TODO find the link if it exists or create and new one
+		#
 		@link = Link.new(link_params)
 		if @link.save
+			TitleWorker.perform_async(@link.given_url)	
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Trailing whitespace detected.